packages/atomic/src/components/common/result-link/attributes-slot.ts

```diff
@@ -5,16 +5,41 @@ export function getAttributesFromLinkSlot(host: HTMLElement, slotName: string) {
   if (!attributesSlot) {
     return;
   }
+  return extractAttributeFromLink(attributesSlot, slotName);
+}
+
+export function getLinkAttributesFromSlotChild(
```

Comment:
Yeah I still need them. Its because the slot is forwarded from `atomic-smart-snippet` to `atomic-smart-snippet-source`. So the actual `<a>` tag is still a children of `atomic-smart-snippet` even if the slot becomes the child of `atomic-smart-snippet-source`. That is why we need to use the `slot.assignedElements()` method to fetch its children

Reply:
Maybe the first iteration was the best? To just forward the attributes through `atomic-smart-snippet-source`?